com.unity.perception/Editor/GroundTruth/PerceptionCameraEditor.cs

```diff
@@ -151,6 +159,37 @@ public override void OnInspectorGUI()
                 GUILayout.EndVertical();
             }
 
+            GUILayout.Space(10);
+
+            var userBaseDir = PlayerPrefs.GetString(SimulationState.userBaseDirectoryKey);
```

Comment:
This path only needs to be set once per project, so it feels like it is using up a lot of space in the inspector. Also it is a project-wide setting. For both those reasons, should this be moved into a project settings window? See https://docs.unity3d.com/2019.1/Documentation/ScriptReference/SettingsProvider.html

With sensors coming soon (tm), PerceptionCamera will no longer be the single interface for dataset generation

Reply:
Interesting. So this gives us the ability to have entries in the actual project settings window of unity?
I agree it would be nice to have this there. I think we should start thinking more seriously about taking some settings out of perception camera (e.g. simulation delta time) and putting them in a more central place once sensors start trickling in.